app/controllers/register-service.controller.js

```diff
@@ -35,60 +40,62 @@ module.exports = {
    * @param req
    * @param res
    */
-  submitRegistration: (req, res) => {
+  submitRegistration: async function submitRegistration (req, res) {
```

Comment:
This method should be hung up on the walls as an example of taking something super complex and making it easy to understand. Really good change 👍